[BUG] Updates needed to the Actions API topic (#1502)
elastic/security-docs: docs/cases/api/actions-api/cases-actions-api-intro.asciidoc

{es-sec} uses these external APIs to send cases:

* {sn}: https://developer.servicenow.com/dev.do#!/reference/api/rome/rest/c_ImportSetAPI[Import Set API]
+

> IMPORTANT: {sn} ITSM and SecOps connectors created in {stack} version 7.15.0 or earlier are marked as deprecated after you upgrade to version 7.16.0 or later. It is strongly recommended to <<post-upgrade-deprecated-sn-connector, update deprecated connectors>> or create new ones to ensure you have access to connector enhancements, such as updating incidents. 

Reviewer: In cases, you can update incidents even with the old API. Only in detections you get this enhancement. Another enhancement is that they can map the data into ServiceNow tables of their choice.

Author: Slacked @cnasikas about this and am changing it to say that ITSM and SecOps connectors created in 7.15 or earlier use the V2 Table API and those created in 7.16+ use the Import Set API.

> IMPORTANT: {sn} ITSM and SecOps connectors created in {stack} version 7.15.0 or earlier use the https://developer.servicenow.com/dev.do#!/reference/api/quebec/rest/c_TableAPI[Table API]. They are marked as deprecated after you upgrade to version 7.16.0 or later. To ensure you have access to connector enhancements, such as updating incidents, we strongly recommended that you <<post-upgrade-deprecated-sn-connector, update deprecated connectors>> or create new ones. 

Reviewer: I want to emphasize that from within Cases you can update incidents with the old connector and prior to 7.16. In 7.16, with the new connectors, we introduced the ability to update incidents from within Rules (prior to 7.16 it was available). I am not sure if this is clear to our users.

{es-sec} uses these external APIs to send cases:

* {sn}: https://developer.servicenow.com/dev.do#!/reference/api/rome/rest/c_ImportSetAPI[Import Set API]
+

> IMPORTANT: {sn} ITSM and SecOps connectors created in {stack} version 7.15.0 or earlier use the https://developer.servicenow.com/dev.do#!/reference/api/quebec/rest/c_TableAPI[Table API]. They are marked as deprecated after you upgrade to version 7.16.0 or later and must be <<post-upgrade-deprecated-sn-connector, updated>> to ensure you have access to new connector enhancements. For example, you can push incident updates from cases that are using connectors created in version 7.15.0 or earlier. However, pushing incident updates from rules is a newer enhancement and you must update your connector or create a new one to use it. 

Reviewer 2: ```suggestion
IMPORTANT: {sn} ITSM and SecOps connectors created in {stack} version 7.15.0 or earlier use the https://developer.servicenow.com/dev.do#!/reference/api/quebec/rest/c_TableAPI[Table API]. They are marked as deprecated after you upgrade to version 7.16.0 or later and must be <<post-upgrade-deprecated-sn-connector, updated>> to ensure you have access to new connector enhancements. For example, you can push incident updates from cases using connectors created in version 7.15.0 or earlier. However, pushing incident updates from rules in {stack} 7.16.0 and later requires you to update your connector or create a new one to use it. 
```

Author: Agree with the first edit, but am leaning towards keeping the last sentence as it is because I think it flows better with the one that comes before it.
